azure-devops/shared/06_pagopa-selc-backoffice-backend.tf

```diff
@@ -0,0 +1,133 @@
+      sonarcloud = {
+        # TODO azure devops terraform provider does not support SonarCloud service endpoint
+        service_connection = "SONARCLOUD-SERVICE-CONN"
+        org                = "pagopa"
+        project_key        = "pagopa_pagopa-selfcare-ms-backoffice-backend"
+        project_name       = "pagopa-selfcare-ms-backoffice-backend"
+      }
+    }
+  }
+}
+
+locals {
+  # global vars
+  pagopa-selc-backoffice-backend-variables = {
+    cache_version_id = "v1"
+    default_branch   = var.pagopa-selc-backoffice-backend.repository.branch_name    
+  }
+  
+  # global secrets
+  pagopa-selc-backoffice-backend-variables_secret = {
+
+  }
+  # code_review vars
+  pagopa-selc-backoffice-backend-variables_code_review = {
+    danger_github_api_token = "skip"
+    sonarcloud_service_conn = var.pagopa-selc-backoffice-backend.pipeline.sonarcloud.service_connection
+    sonarcloud_org          = var.pagopa-selc-backoffice-backend.pipeline.sonarcloud.org
+    sonarcloud_project_key  = var.pagopa-selc-backoffice-backend.pipeline.sonarcloud.project_key
+    sonarcloud_project_name = var.pagopa-selc-backoffice-backend.pipeline.sonarcloud.project_name
+  }
+  # code_review secrets
+  pagopa-selc-backoffice-backend-variables_secret_code_review = {
+  }
+  # deploy vars
+  pagopa-selc-backoffice-backend-variables_deploy = {
+    git_mail          = module.secrets.values["azure-devops-github-EMAIL"].value
+    git_username      = module.secrets.values["azure-devops-github-USERNAME"].value
+    github_connection = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_github_rw_name
+    tenant_id         = module.secrets.values["TENANTID"].value
+    # acr section
+    image_repository_name = replace(var.pagopa-selc-backoffice-backend.repository.name, "-", "")
+    dev_container_registry_service_conn = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_dev_id
+    # uat_container_registry_service_conn  = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_uat_id
+    # prod_container_registry_service_conn = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_prod_id
+
+    # aks section
+    k8s_namespace               = "shared"
+    dev_kubernetes_service_conn = azuredevops_serviceendpoint_kubernetes.aks_dev.id
+    # uat_kubernetes_service_conn  = azuredevops_serviceendpoint_kubernetes.aks_uat.id
+    # prod_kubernetes_service_conn = azuredevops_serviceendpoint_kubernetes.aks_prod.id
+
+    #dev_container_registry_name
+    dev_container_namespace = "pagopadcommonacr.azurecr.io"
+    #uat_container_registry_name
+    # uat_container_namespace  = "pagopaucommonacr.azurecr.io"
+    #prod_container_registry_name
+    # prod_container_namespace = "pagopapcommonacr.azurecr.io"
+  }
+  
+  # deploy secrets
+  pagopa-selc-backoffice-backend-variables_secret_deploy = {
+
+  }
+}
+
+module "pagopa-selc-backoffice-backend_code_review" {
```

Comment:
Selfare do you need CI pipeline too ?

Reply:
selfcare need both code review and deploy